keyboards/vitamins_included/rev1/rules.mk

```diff
@@ -1,5 +1,8 @@
-BACKLIGHT_ENABLE = no
-AUDIO_ENABLE = yes
-RGBLIGHT_ENABLE = yes
-DEBUG_ENABLE = no
-CONSOLE_ENABLE = no
+QUANTUM_LIB_SRC += matrix.c \
```

Comment:
To clarify, not everything needs to be in `QUANTUM_LIB_SRC`.  

Specifically, it should be broken up like: 

```c
SRC += matrix.c \
	   split_util.c \
	   ssd1306.c

QUANTUM_LIB_SRC += i2c.c \
	   serial.c
```

This way, the `SRC` files can be optimized, but the `QUANTUM_LIB_SRC` files won't be.  

Sorry for the confusion!

Reply:
Thanks for the help! Let's just say I was easily confused yesterday :)